ynr/apps/bulk_adding/models.py

```diff
@@ -1,5 +1,5 @@
-from django.contrib.postgres.fields import JSONField
 from django.db import models
+from django.db.models import JSONField
```

Comment:
I think these changes will require an `AlterField` change to be generated with `makemigrations`